app/src/main/res/values-de/strings.xml

```diff
@@ -26,7 +26,7 @@
     <string name="planted_date">%1$s gepflanzt in %2$s</string>
     <string name="menu_item_share_plant">Teilen</string>
     <string name="share_text_plant">Schau dir die %s Pflanze in der App "Sunflower" an</string>
-    <string name="remove_plant_title">Pflanze(n) entfernen</string>
+    <string name="remove_plant_title">Pflanze (n) entfernen</string>
```

Comment:
I think in German, people still like using without empty `Pflanze(n)`, I feel at least very strange to see empty there.

Reply:
Fixed. Yeah, I .. don't know why I trusted Google Translate for that ;) Thanks!